UltiSnips/elixir.snippets

```diff
@@ -0,0 +1,126 @@
+snippet if
+if ${1:condition} do
+	${2:expression}
+${3:end}
+endsnippet
+
+snippet unless
+unless ${1:condition} do
+	${2:expression}
+${3:end}
+endsnippet
+
+snippet else
+else
+	${1:expression}
+end
+endsnippet
+
+snippet iff
```

Comment:
I prefer not to bind things to hitting the same key twice in a row, for some reason I find this harder to type than two different keys.

I do think this is better though- `:` requires two keys to type.

Both the snippets format and ultisnips format snippets should have the same trigger though.